test/node/replumb/require_node_test.cljs

```diff
@@ -446,7 +561,15 @@ trim-newline
     (process-require)
     (process-goog-import)
     (ns-macro)
-    (ns-macro-with-require-macro)
+
+    (ns-macro-require-refer-macros)
+    (ns-macro-require-include-macros)
+    (ns-macro-require-macros)
+    (ns-macro-require-macros-refer)
+    (ns-macro-use-macros)
+    (ns-macro-require)
+    (ns-macro-require-macros-as)
```

Comment:
As said, the order is fixed but the results of the tests are true, I double checked in the repl.
I will open an issue for that.